### PR TITLE
add missing UnZip dependency for Python 3.8.2

### PR DESCRIPTION
**Pull request by terjekv**
(created using `eb --new-pr`)

requires ~~#11457~~

### CODE REVIEW

**Comment by terjekv** [comment]
Also fixes the issue here: https://github.com/easybuilders/easybuild-easyconfigs/pull/11417#issuecomment-705979646

**Comment by terjekv** [comment]
Test report by @terjekv
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
ninhursaga.uio.no - Linux RHEL 8.2, x86_64, Intel(R) Xeon(R) Silver 4216 CPU @ 2.10GHz (cascadelake), Python 3.6.8
See https://gist.github.com/dffbad9d8e325e43c8401d50d4e1fd83 for a full test report.

**Comment by terjekv** [comment]
Test report by @terjekv
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
ninhursaga.uio.no - Linux RHEL 8.2, x86_64, Intel(R) Xeon(R) Silver 4216 CPU @ 2.10GHz (cascadelake), Python 3.6.8
See https://gist.github.com/a912e35ddfb2210b0fe5da5557e57ef4 for a full test report.

File: easybuild/easyconfigs/p/Python/Python-3.8.2-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -20,6 +20,7 @@ dependencies = [
     ('ncurses', '6.2'),
     ('SQLite', '3.31.1'),
     ('XZ', '5.2.5'),
+    ('UnZip', '6.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-3.8.2-GCCcore-9.3.0.eb:23]: @terjekv This is only a build dep, or am I overlooking something?

See also #10427

Comment (terjekv) [reply easybuild/easyconfigs/p/Python/Python-3.8.2-GCCcore-9.3.0.eb]: Correct, my bad.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node3134.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/8e16c73df6fce4f6acfbf07c50d3c7e9 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @terjekv!

**Pull request merged**